testsuite/features/step_definitions/command_steps.rb

```diff
@@ -175,7 +175,7 @@
     package.gsub! "suma", "uyuni"
   end
   node = get_target(host)
-  if host.include? 'ubuntu'
+  if db_host?(host)
```

Comment:
Where do you extract the db_host variable? I think this refactor is dangerous and might be syntactically wrong even. Instead of querying for whatever is present and saving it to host variable and comparing that with each value for each host, you are changing it to save each different value to a different variable named for each type of host. That could be dangerous if we try to use any of these variables in a place where they are not initialised.

Reply:
it's not a variable, it's a function, defined in `commonlib.rb`.